addons/mail/static/src/xml/thread.xml

```diff
@@ -20,10 +20,16 @@
                     <i class="fa fa-picture-o mr8" t-if="widget.activeAttachment.fileType == 'image'" role="img" aria-label="Image" title="Image"/>
                     <i class="fa fa-file-text mr8" t-if="widget.activeAttachment.fileType == 'application/pdf'" role="img" aria-label="PDF file" title="PDF file"/>
                     <i class="fa fa-video-camera mr8" t-if="widget.activeAttachment.fileType == 'video'" role="img" aria-label="Video" title="Video"/>
-                    <t t-esc="widget.activeAttachment.name"/>
-                    <a role="button" href="#" class="o_download_btn ml8 small" data-toggle="tooltip" data-placement="right" title="Download"><i class="fa fa-fw fa-download" role="img" aria-label="Download"/></a>
+                    <span class="o_viewer_document_name" t-esc="widget.activeAttachment.name"/>
                 </span>
-                <a role="button" class="o_close_btn float-right" href="#" aria-label="Close" title="Close">×</a>
+                <div class="o-autogrow"/>
+                <a class="o_download_btn o_document_viewer_topbar_button btn" data-toggle="tooltip" title="Download">
```

Comment:
This `data-toggle="tooltip"` makes a stylized tooltip on hover, instead of the standard browser title display.

This is currently present only on Download And Split PDF in old code. This is missing from the close button (even in the old code) and we don't have it anywhere in the new code.

The lack of consistency is a bit strange. Do want it or not?

Reply:
I would not use `data-toggle` in new code. If we really want stylized tooltip, it's best doing so without bootstrap.

When I looked the style with `data-toggle`, it's not that nice, and the direction is at the appreciation of boostrap. To the point it goes below for "Split PDF" but left for "Download" (which hides the "Split PDF" button). The white color doesn't match the somewhat dark theme of the viewer.